demo

```diff
@@ -216,6 +216,7 @@ start_server() {
             -f docker-compose.yml \
             -f docker-compose.storage.minio.yml \
             -f docker-compose.demo.yml \
+            -f docker-compose.azure.yml \
```

Comment:
Regarding the demo: Does this require extra steps to work, or is it just an additional component that is offered, but won't do anything unless you actively engage with it?